.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,91 @@
+on: [push]
+name: Build python wheels
+
+
+jobs:
+
+  build_source:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install python
+        uses: actions/setup-python@v1
```

Comment:
Should we pin the version of python here?

Reply:
Yes, we should :) thanks for pointing it out, I missed it.

GitHub Actions documentations says [here](https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#using-the-default-python-version):

> We recommend using setup-python to configure the version of Python used in your workflows because it helps make your dependencies explicit.